packages/ra-ui-materialui/src/list/BulkActionsToolbar.js

```diff
@@ -59,6 +63,14 @@ const BulkActionsToolbar = ({
             })}
             {...sanitizeListRestProps(rest)}
         >
+            <IconButton
```

Comment:
I find it very weird to propose to "unselect" *before* showing the number of selected items. I'd expect the contrary.

Reply:
It's the way of Google Photos. It makes sense to me personally. That's why I copied it.
![image](https://user-images.githubusercontent.com/465989/68838893-d9c04c80-06bf-11ea-9068-b6026872cf8f.png)

Reply:
OK, I get it, thanks